### ISSUE TITLE
Prompt Enhancement

### ISSUE THREAD

**Issue by ChenyiZhang007** [XiaRho/SEMat#6]
Is Prompt Enhancement referring to the process of encoding a bbox into a binary mask and then adding its embedding to the image embedding? I couldn't find a clear description of Prompt Enhancement in the paper. By the way, is there a plan to release the training code?

**Comment by XiaRho** [comment]
Yes, you can refer to line 111-120 in `modeling/semantic_enhanced_matting/modeling/image_encoder.py` for details. The training code is considered to be released after the paper is accepted, but as the work was done during my internship, the code ownership is with the company. There might be restrictions on releasing it.

**Comment by ChenyiZhang007** [comment]
Thank you for your response. I have one more question regarding the first row of results in Table 6(b). Does this version of the model exclude the adaptor and matting tokens? In other words, does the matting decoder only receive the image and SAM features without the guidance of the trimap?

**Comment by ChenyiZhang007** [comment]
Thank you for your response
